File: module-server/src/main/java/de/twomartens/wahlrecht/model/dto/v1/ElectedCandidates.java
package de.twomartens.wahlrecht.model.dto.v1;

import java.util.LinkedList;
import java.util.Map;

public record ElectedCandidates(ElectedResult overallResult,
                                Map<Integer, ElectedResult> constituencyResults,
                                LinkedList<Double> electionNumbersForSeatAllocation) {

}

File: module-server/src/main/kotlin/de/twomartens/wahlrecht/model/dto/v1/ElectedCandidates.kt
package de.twomartens.wahlrecht.model.dto.v1

data class ElectedCandidates(val overallResult: ElectedResult,
                             val constituencyResults: Map<Int, ElectedResult>,
                             val electionNumbersForSeatAllocation: List<Double>,
                             val seatAllocation: Map<String, Int>)
